ignite/templates/app/stargate/app/app.go.plush

```diff
@@ -101,10 +101,8 @@ import (
 	tmos "github.com/tendermint/tendermint/libs/os"
 	dbm "github.com/tendermint/tm-db"
 
-	"github.com/ignite/cli/ignite/pkg/cosmoscmd"
-	"github.com/ignite/cli/ignite/pkg/openapiconsole"
-
 	"<%= ModulePath %>/docs"
+	"<%= ModulePath %>/cmd/<%= BinaryNamePrefix %>d/cmd"
```

Comment:
I replicated the gaia repository where the related code is located in `cmd/gaiad/cmd`